gtsam/discrete/DiscreteKey.h

```diff
@@ -69,6 +69,13 @@ namespace gtsam {
       push_back(key);
       return *this;
     }
+
+    /// Return the DiscreteKey vector as a set.
+    std::set<DiscreteKey> asSet() const {
```

Comment:
prefer not to have changes outside hybrid. Make static function in cpp file where you need it?

Reply:
prefer not to have changes outside hybrid. Make static function in cpp file where you need it?